lab4.py

```diff
@@ -0,0 +1,61 @@
+class Student:
+   
+    def __init__(self, name="Невідомий", rating=0, height=0):
+        self.__name = name         
+        self.__rating = rating     
+        self.__height = height     
+    
+    def get_name(self):
```

Comment:
your code is good, you can use @property if you want

Reply:
thank you